script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -939,52 +1092,61 @@ The girl's single thrust could be seen as the roar of her soul.
 [sha:9ca18d42de70a0895ed6fecaf1114332f51034b7]{
 -- Page 346, Offset 36283.
 -- それを甘んじて受けたいと想う<情|じょう>ごと、彼女は敵を粉砕した。
-Along with her desire to accept those feelings wholeheartedly, she lays waste to her foe.
+// Assuming this is like 甘んじて受け入れる?
+// https://www.weblio.jp/content/%E7%94%98%E3%82%93%E3%81%98%E3%81%A6%E5%8F%97%E3%81%91%E5%85%A5%E3%82%8C%E3%82%8B
+// As in, Ciel doesn't like that she has to do this?
+Wishing she could resign herself to what she is about to do,
+ the Executor lays waste to her foe.
```

Comment:
Uhhh, well, I'll just translate it a bit more analytically

>[Ciel fires], and blows it——alongside her own feelings of wanting to just take it head on——apart.


Here's a more normal TL
>Ciel takes aim, at her enemy, and at her own feelings of wanting to just submissively take the thrust head on, blowing both of them apart.